scripts/override_os_images.py

```diff
@@ -18,6 +18,9 @@ def get_os_image(os_images, ocp_version, cpu_architecture="x86_64"):
 
 
 def extract_version(release_image: str) -> str:
+    if not release_image:
+        return ""
```

Comment:
what will happen if we will return an empty string? Are we going to continue the installation with the default version?

Reply:
Empty string will behave the same as not exporting the env-var
Yes, it will go with the default